src/ddmd/declaration.d

```diff
@@ -907,7 +907,7 @@ extern (C++) class VarDeclaration : Declaration
         {
             if (!type && !_init)
             {
-                printf("VarDeclaration('%s')\n", id.toChars());
+                //printf("VarDeclaration('%s')\n", id.toChars());
```

Comment:
Did you grep for any others?

Shouldn't there be an ICE here instead of a noop?

Reply:
I commented it out because the compiler did not build. I assume this is an accidental leftover debug output.